### PR TITLE
Use correct CPU count, new SORT_RAM variable

### PR DESCRIPTION
**Pull request by reineckef**
Use the correctly calculated number of CPUs for the call to Bowtie2.

### CODE REVIEW

**Comment by reineckef** [comment]
Hi Nicolas,

Thank you for this great pipeline. I have used it, and I like it a lot. I found one little issue with the CPU count. You have correctly calculated that, but used the variable for the command line.

The second thing that is different here is a SORT_RAM variable added to the configuration. We have had some issues with samtools sort using all memory and getting killed, especially if run on many cores on a system (not using cluster mode). This variable will help us to tune samtools sort, which will use temp-files and merging if granted RAM is not sufficient.

Thank you,
Frank

File: config-hicpro.txt [hunk 1]
```diff
@@ -14,6 +14,7 @@ RAW_DIR = rawdata
 ## SYSTEM AND SCHEDULER - Start Editing Here !!
 #######################################################################
 N_CPU = 2
+SORT_RAM = 768M
```
[the remark this answers is not in the document]
Comment (reineckef) [reply config-hicpro.txt]: We have built docker containers with your pipeline (and some wrappers to run and post-process) and we deploy these on different hardware, both locally and on cloud systems. The configuration (RAM and CPU) can change for every job depending on where we run it. So, the config file is the best option to specify these things, and leaving the default to be the same for samtools sort does not change anything unless someone edits the config.

Thanks!

**Comment by reineckef** [comment]
Hi Nicolas,

I have updated the devel in my fork for the -S 90**%** change (pull request #207), and did the same in my master. Now, that has automatically become a part of this much older pull request #203, due to automatic update. Sorry for any disturbance caused by this duplication.

Cheers,
Frank

**Comment by nservant** [comment]
Hi Frank,
I'm sorry, but I cannot merge because github ask me to modify the master.
Could you make a new pull request from the devel ?
And close this one ?
Sorry for this.
N

**Comment by reineckef** [comment]
Create this again for the devel branch

**Pull request closed without merging**